android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUILabel.java

```diff
@@ -276,6 +286,11 @@ public void propertyChanged(String key, Object oldValue, Object newValue, KrollP
 		} else if (key.equals(TiC.PROPERTY_SHADOW_COLOR)) {
 			shadowColor = TiConvert.toColor(TiConvert.toString(newValue));
 			tv.setShadowLayer(shadowRadius, shadowX, shadowY, shadowColor);
+		} else if (key.equals(TiC.PROPERTY_ATTRIBUTED_STRING) && newValue instanceof AttributedStringProxy) {
```

Comment:
We should try to resolve the parity issue if we can.

| Neeraj Gupta | SVP, Product and Engineering
| Appcelerator, Inc | e: ngupta@appcelerator.com<mailto:%20ngupta@appcelerator.com> | o:+1 650-528-2931 |
| 440 N. Bernardo Ave, Mtn View, CA 94043

From: pingwang2011 <notifications@github.com<mailto:notifications@github.com>>
Reply-To: appcelerator/titanium_mobile <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Monday, January 5, 2015 at 2:57 PM
To: appcelerator/titanium_mobile <titanium_mobile@noreply.github.com<mailto:titanium_mobile@noreply.github.com>>
Subject: Re: [titanium_mobile] [TIMOB-18062] iOS: AttributedString Parity with Android (#6463)


In android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUILabel.java<https://github.com/appcelerator/titanium_mobile/pull/6463#discussion-diff-22496623>:

> @@ -276,6 +286,11 @@ public void propertyChanged(String key, Object oldValue, Object newValue, KrollP
>               } else if (key.equals(TiC.PROPERTY_SHADOW_COLOR)) {
>                       shadowColor = TiConvert.toColor(TiConvert.toString(newValue));
>                       tv.setShadowLayer(shadowRadius, shadowX, shadowY, shadowColor);
> +             } else if (key.equals(TiC.PROPERTY_ATTRIBUTED_STRING) && newValue instanceof AttributedStringProxy) {


@bhatfield<https://github.com/bhatfield>, @vishalduggal<https://github.com/vishalduggal>, should we resolve this parity issue or just document it?

-
Reply to this email directly or view it on GitHub<https://github.com/appcelerator/titanium_mobile/pull/6463/files#r22496623>.